client/src/Dashboard/Dashboard.js

```diff
@@ -107,89 +158,79 @@ export function Dashboard() {
-      minRevenue: '$4000.90',
-      maxRevenue: '$5000.80'
-    },
-    {
-      key: '6',
-      childName: 'Kathryn Murphy',
-      caseNumber: '295340',
-      business: 'Austin Community Child Care',
-      attendanceRate: '50%',
-      minRevenue: '$1240.90',
-      maxRevenue: '$2000.50'
-    },
-    {
-      key: '7',
-      childName: 'Leslie Alexander',
-      caseNumber: '803162',
-      business: 'Ravenswood Daycare',
-      attendanceRate: '52%',
-      minRevenue: '$1109.90',
-      maxRevenue: '$2188'
+  useEffect(() => {
+    const responseValue = async () => {
+      const businesses = await makeRequest({
+        type: 'get',
+        url: '/api/v1/businesses',
+        headers: { Authorization: token }
+      })
+      const allBusinesses = await businesses.json()
+      if (!allBusinesses.error) {
+        setBusinessList(allBusinesses)
+      }
     }
-  ]
+
+    const getDashboardData = async () => {
+      const response = await makeRequest({
+        type: 'get',
+        url: '/api/v1/case_list_for_dashboard',
+        headers: { Authorization: token }
+      })
+      const parsedResponse = await response.json()
+
+      if (!parsedResponse.error) {
+        const data = parsedResponse.reduce((acc, cv, index) => {
+          const {
+            full_name: name = '',
+            approvals: [{ case_number: caseNumber = '' }],
+            business: { name: businessName = '' }
+          } = cv
+
+          return [
+            ...acc,
+            {
+              key: index,
+              childName: name,
+              caseNumber: caseNumber,
+              business: businessName,
+              // these values will be updated as the case_list endpoint is updated
+              attendanceRate: '',
+              minRevenue: '',
+              maxRevenue: '',
+              potentialRevenue: ''
+            }
+          ]
+        }, [])
+
+        setDashboardData(data)
+      }
+    }
+    // Interesting re: refresh tokens - https://github.com/waiting-for-dev/devise-jwt/issues/7#issuecomment-322115576
+    responseValue()
+    getDashboardData()
+    // still haven't found a better way around this - sometimes we really do
+    // only want the useEffect to fire on the first component load
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
 
   return (
-    <div className="dashboard">
+    <div className="dashboard sm:mx-8">
       <div className="m-2">
         <Typography.Title>{t('dashboardTitle')}</Typography.Title>
         <Typography.Text className="md-3">
           {t('revenueProjections')}
         </Typography.Text>
       </div>
       <Table
-        dataSource={staticData}
+        dataSource={dashboardData}
         columns={columns}
         bordered={true}
         size={'medium'}
         pagination={false}
         sticky
         className="dashboard-table"
+        scroll={{ x: 'max-content' }}
       >
         {businessList &&
           businessList.map(business => {
```

Comment:
See the other comment, is this even rendering anymore?

Reply:
yeah, definitely don't need this! deleting.